VaticanBAV/borget/BAVborget24.xml

```diff
@@ -0,0 +1,540 @@
+                        </msItem>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.4">
+                        <incipit xml:lang="gez">በስመ፡ አብ፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> በብርሃን፡ ስሙ፡ ለእግዚአብሔር፡ ኢይትፈለጥ፡ ሰማይ፡ ወምድር፡ ፯መላእክት፡ ብርሃናውያን፡ እለ፡ ይነብሩ፡ ታሕተ፡ መንጦላዕቱ፡ ቅዱስ፡</incipit>
+
+                        <msItem xml:id="ms_i1.4.1">
+                          <title type="incomplete" ref="LIT1758Lefafa#FourthChapter"/>
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.4.2">
+                          <title type="complete" ref="LIT1758Lefafa#FifthChapter"/>
+                        </msItem>
+
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.5">
+                        <title type="complete" ref="LIT1758Lefafa#SixthChapter"/>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.6">
+                        <title type="complete" ref="LIT1758Lefafa#ThirdChapter"/>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.7">
+                        <title type="incomplete" ref="LIT1873Mangad#Sixth"/>
+                      </msItem>
+
+                    </msItem>
+
+                  </msContents>
+
+                  <physDesc>
+                    <objectDesc form="Scroll">
+                      <supportDesc>
+                        <support>
+                          <material key="parchment"/>
+                        </support>
+                        <extent>
+                          <note>Consists of <measure unit="strips">3</measure> strips sewn together.</note>
+                          <dimensions type="outer" unit="mm">
+                            <width>72</width>
+                            <height>1700</height>
+                          </dimensions>
+                        </extent>
+                      </supportDesc>
+                      <layoutDesc>
+                        <layout columns="1" writtenLines="306"/>
+                      </layoutDesc>
+                    </objectDesc>
+                    <decoDesc>
+                      <decoNote xml:id="d1" type="miniature">
+                        <desc>Three black and red <term key="cross"/> with interlaced border above and below.</desc>
+                      </decoNote>
+                      <decoNote xml:id="d2" type="miniature">
+                        <desc>Three black and red <term key="cross"/> with decorative border above and below.</desc>
+                      </decoNote>
+                      <decoNote xml:id="d3" type="miniature">
+                        <desc>Three black and red <term key="cross"/> with interlaced border with eyes above and below.</desc>
+                      </decoNote>
+                    </decoDesc>
+                    <additions>
+                      <list>
+                        <item xml:id="e1" corresp="#d1">
```

Comment:
same as for decoNotes, some sort of at least relative location is needed to disambiguate, or what we see is 
```
Extras
1 The words ቄል, ኤል, and ሄል are written between three crosses.

2 The words ቄል, ኤል, and ሄል are written between three crosses.

3 The words ቄል, ኤል, and ሄል are written between three crosses.
```